modules/console/pages/config/security/authentication.adoc

```diff
@@ -192,6 +418,75 @@ authentication:
     enabled: true # <3>
 ----
 
+--
+Kubernetes Embedded::
++
+--
+
+When Redpanda Console is part of the Redpanda Helm chart or Operator:
+
+[tabs]
+====
+Operator::
++
+.`redpanda-cluster`.yaml
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha2
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  clusterSpec:
+    console:
+      enabled: true
+      console:
+        config:
+          authentication:
+            jwtSigningKey: "<secret-key>" # <1>
+            useSecureCookies: true # <2>
+            basic:
+              enabled: true # <3>
+----
+
+Helm::
++
+.`redpanda-values.yaml`
+[,yaml]
+----
+console:
+  enabled: true
+  console:
+    config:
+      authentication:
+        jwtSigningKey: "<secret-key>" # <1>
+        useSecureCookies: true # <2>
+        basic:
+          enabled: true # <3>
+----
+====
+
+--
+Kubernetes Standalone::
```

Comment:
```suggestion
Kubernetes standalone::
```
Maybe search all files to make sure these tab names are consistent

Reply:
Looks like you found the only instances 😄